apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/brevbaker/Brevkoder.kt

```diff
@@ -60,6 +61,7 @@ enum class Brevkoder(val redigering: EtterlatteBrevKode, val ferdigstilling: Ett
     BP_OPPHOER(BARNEPENSJON_OPPHOER_UTFALL, BARNEPENSJON_OPPHOER),
     BP_REVURDERING(BARNEPENSJON_REVURDERING_UTFALL, BARNEPENSJON_REVURDERING),
     BP_VARSEL(BARNEPENSJON_VARSEL_UTFALL, BARNEPENSJON_VARSEL),
+    BP_INFORMASJON_DOEDSFALL(BARNEPENSJON_INFORMASJON_DOEDSFALL, TOM_MAL_INFORMASJONSBREV),
```

Comment:
Jeg henger ikke heelt med på hva som blir rett å sette for `ferdigstilling` for brev som kun er automatiske?

Reply:
Litt enkelt sagt er den første malen her den redigerbare delen og den andre er ikkje-redigerbar. Så viss det er viktig at teksten ikkje er redigerbar, bør han flyttast til ferdigstillingsmalen.

Men trur det er verdt å stille eit par ekstra spørsmål om det er _så farleg_ at han ikkje kan redigerast, for det er sånn som du har gjort det her er den modellen vi har lagt opp til elles i systemet per no.

Reply:
Okey, takk for oppklaring. Da beholder jeg det slik jeg hadde tenkt 👍. Det er ikke egentlig vesentlig at den ikke kan redigeres, men akkurat nå så trigges den fra en automatisk jobb hvor det ikke er noen steg som kan føre til manuelt innhold. Men det kan jo være at det kommer et ønske om å benytte samme mal manuelt senere.